#141409267 brief tour for effective onboarding

johnkingzy opened the pull request:
#### What does this PR do?
A brief tour of how the game is played and the important controls#### Description of Task to be completed?

#### How should this be manually tested?
Once a user just signup he will be taken on a brief tour of the gam

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
Users should be quickly and effectively onboarded onto the app

#### Screenshots (if appropriate)
N/A

#### Questions:

Commit:
feature(design): design
brief tour for users to quickly and effectively onboarded onto the app
starts[#141409267]

Commit:
Just updated my branch with the staging branch
Merge branch 'staging' of https://github.com/andela/might-guy-cfh into feature/141409267/effective-onboarding

Commit:
feature(design): design
brief tour for users to quickly and effectively onboarded onto the app
finishes[#141409267]

Commit:
feature(design):
brief tour for effective onboarding
Finishes [#141409267]

uopaluwa reviewed (approved):
Improve your commit messages.

Commit:
feature(addFriend): add
implement add friend for the invite friends feature
[Start #141409277]

Commit:
feature(effective-onboarding): brief tour
use bower to install all neccessary library for the game tour
Finishes [#141409267]

Commit:
feature(effective-onboarding): implement
brief tour for effective onboarding
Finishes[141409267]

uopaluwa commented:
Not up to date.
